packages/api/src/domain/invalid-links.ts

```diff
@@ -0,0 +1,15 @@
+import { BaseDomain, BaseDomainCreate } from "@metriport/core/domain/base-domain";
+import { CQLink } from "../external/carequality/cq-patient-data";
+import { CwLink } from "../external/commonwell/cw-patient-data";
+
+export type InvalidLinksData = {
+  carequality?: CQLink[];
+  commonwell?: CwLink[];
```

Comment:
nit: CW

Reply:
CQLink should actually be Cq based on our engineering standards. I can update this now but then its going to add a lot of new file changes since its used in a bunch of places

Reply:
Yeah, CamelCase, even for acronyms, please